ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -345,6 +362,68 @@ public static BlobProperties buildBlobProperties(Map<String, Object> args)
   }
 
   /**
+   * Build a {@link ByteRange} given the arguments associated with a request.
+   * @param args the arguments associated with the request.
```

Comment:
it is a map of all headers and query string arguments. I will add more details to the description.